source/deployment/guest-accounts.rst

```diff
@@ -3,63 +3,101 @@
 
-Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__. 
+Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__.
 
-Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization. The Guest Account feature controls access to channels and other members within the organization. 
+Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization.
+The Guest Account feature controls access to channels and other members within the organization.
 
-An organization may use guest accounts to work on a website project with resources from external design firm. Guest Accounts could also be used to collaborate with customers on a support issue.
+For example, an organization may use Guest Accounts to work on a website project with resources from an external design firm. Guest Accounts
+could also be used to collaborate with customers on a support issue.
 
-Guests can: 
+Guests can:
 
 - Pin messages to channels
 - Use slash commands (excluding restricted commands such as invite members, rename channels, change headers, etc)
 - Favorite channels
 - Mute channels
 - Update their account settings
 
-Guests cannot: 
+Guests cannot:
 
 - Discover public channels
 - Join open teams
 - Create direct messages or group messages with members who aren’t within the same channel
+- Invite people
 
 
 Enabling Guest Accounts
 ------------------------
 
 1. Navigate to **System Console > Authentication > Guest Access**.
 2. Set **Enable Guest Access** to ``True``.
-3. (Optional) **Whitelist domains that are acceptable for Guest Access**.  
+3. (Optional) **Whitelist domains that are acceptable for Guest Access**.
  - This allows the System Administrators to set a list of approved guest domains. If you have team domain restrictions, you will also need to add your guest domain to **Team Settings > Allow only users with a specific email domain to join this team**.
 4. (Optional) **Enforce Multi-Factor Authentication (MFA) for your Guests**.
- - If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users. 
+ - If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users.
 
 Guest Authentication
------------------------------
-Guests can access the Mattermost server via email authentication. Please ensure you enable email invites from **System Console > Signup > Enable Email Invitations**.   
+---------------------
 
-SAML 2.0 and AD/LDAP authentication will be provided in a future release.
+Guests can access the Mattermost server via email invitation, and be authenticated using AD/LDAP or SAML 2.0 with the **Guest Attribute** setting in the
```

Comment:
```suggestion
Guests can access the Mattermost server via email invitation, and be authenticated using AD/LDAP (**Guest Filter**) or SAML 2.0 (**Guest Attribute**).
```

Reply:
This works better, but wondering if we should follow the format of **System Console > AD/LDAP > Guest Filter** and **System Console > SAML 2.0 > Guest Attribute**